### DISCUSSION TITLE
Changes in components don't reflect in playground anymore

### DISCUSSION THREAD

**Discussion by mariusa** [cdmoro/bootstrap-vue-3#551] [General]
playground is good for testing during library development. But now I made a change to `BAvatar` (#550) and playground doesn't reflect it. Avatar keeps showing `false false`.

Is this by design or I'm doing something wrong?

**Comment by VividLemon** [comment]
The playground requires a rebuild of the main package. There's no real way around that. If you're doing development on specific components, you're better off using the main packages App.vue to get hot reloading.

 I prefer to either not include my changes to git, or to discard changes after I'm done. 
 
You could possibly ask Vite to see if there's any way to allow for a hot-reloading type environment in a different package, but I doubt there's any way to do that.

**Comment by aceofwings** [comment]
We will have to write a custom plugin or take advantage of vite's create app api to address this issue.

**Reply by VividLemon** [reply]
I don't imagine either of those are very simple 🤣 I just add single components that I'm working on into the main packages app.vue, and just don't include it in commits. While, when observing _issues_, I tend to view the playground and the behavior of many components.

**Reply by aceofwings** [reply]
I agree, a problem for another day.
